### ISSUE TITLE
Is SmartStoreNET Fork of Nopcommerce?

### ISSUE THREAD

**Issue by ecommerce-tester** [smartstoreag/SmartStoreNET#1511]
If Yes
What version of Nopcommerce?
And which  is the better SmartStoreNET vs NopCommerce?

**Comment by xcellenceit** [comment]
Yes, you can check it [here](https://www.nopcommerce.com/en/boards/topic/24110/smartstorenet/page/3#101044).
